typescript/test/test_application/test_js.ts

```diff
@@ -51,38 +53,48 @@ const JS_TEST_APPLICATION = path.join(
 );
 describe('TestApplication', () => {
   describe('Test application for js users', () => {
-    it('Application test using generated showcase library.', function() {
-      this.timeout(120000);
+    it('npm install showcase', async function() {
+      this.timeout(60000);
       // copy protos to generated client library and copy test application to local.
       fs.copySync(PROTOS, path.join(SHOWCASE_LIB, 'protos'));
       fs.copySync(JS_TEST_APPLICATION, LOCAL_JS_APPLICTION);
       process.chdir(SHOWCASE_LIB);
       try {
-        execSync(`npm install`);
+        await exec(`npm install`);
       } catch (err) {
         console.warn(`Failed to install packages.`);
```

Comment:
Shouldn't we just fail in case of any exception? This code will keep working. Maybe just remove all `try...catch` and let it fail?